src/app/store/students/students.selectors.ts

```diff
@@ -7,6 +7,7 @@ export const selectIsStudentsLoading = createSelector(selectStudentsState, stude
 export const selectIsStudentsLoaded = createSelector(selectStudentsState, studentsState => studentsState.loaded);
 export const selectHasStudentAddFailed = createSelector(selectStudentsState, studentsState => studentsState.hasStudentAddFailed);
 export const selectStudentsError = createSelector(selectStudentsState, studentsState => studentsState.error);
+export const selectHasStudentLoadFailed = createSelector(selectStudentsState, studentsState => studentsState.hasStudentLoadFailed);
```

Comment:
Selector naming could be improved: every selector should start with "get". For example:
```suggestion
export const getHasStudentLoadFailed = createSelector(selectStudentsState, studentsState => studentsState.hasStudentLoadFailed);
```
Should also be applied to other selectors :)